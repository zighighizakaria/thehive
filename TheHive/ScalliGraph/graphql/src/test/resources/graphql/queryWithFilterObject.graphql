query filterPerson {
    allPeople {
        filter(
            filter: {
                name_contains: "o"
            }
        ) {
            sort(
                sort:{
                    age: incr
                }) {
                age {
                    toList
                }
            }
        }
    }
}