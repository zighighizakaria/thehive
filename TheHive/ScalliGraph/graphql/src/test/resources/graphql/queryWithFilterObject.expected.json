{
  "data": {
    "allPeople": {
      "filter": {
        "sort": {
          "age": {
            "toList": [
              29,
              32
            ]
          }
        }
      }
    }
  }
}